test/generated-config.test.ts

```diff
@@ -33,12 +31,12 @@ describe('Generated Config (QuickType)', () => {
 
       const result = Convert.toGitProxyConfig(JSON.stringify(validConfig));
 
-      expect(result).to.be.an('object');
-      expect(result.proxyUrl).to.equal('https://proxy.example.com');
-      expect(result.cookieSecret).to.equal('test-secret');
-      expect(result.authorisedList).to.be.an('array');
-      expect(result.authentication).to.be.an('array');
-      expect(result.sink).to.be.an('array');
+      expect(result).toBeTypeOf('object');
+      expect(result.proxyUrl).toBe('https://proxy.example.com');
+      expect(result.cookieSecret).toBe('test-secret');
+      expect(Array.isArray(result.authorisedList)).toBe(true);
```

Comment:
Theres an assertion for this: https://vitest.dev/api/assert.html#isarray

```suggestion
      assert.isArray(result.authorisedList);
```